### PR TITLE
[JBTM-3753] Make the concurrency of the LRA coordinator start method

### PR DESCRIPTION
**Pull request by marcosgopen**
https://issues.redhat.com/browse/JBTM-3753
Make the concurrency of the LRA coordinator start method (added the microprofile-config.properties)
Added a test in lra-test-basic
Added a README.md in LRA module

JDK17
CORE !TOMCAT AS_TESTS RTS !JACOCO !XTS !QA_JTA !QA_JTS_OPENJDKORB !PERF LRA !DB_TESTS !mysql !db2 !postgres !oracle

### CODE REVIEW

**Comment by marcosgopen** [comment]
I noticed there isn't a README.md for LRA module in Narayana. I think it is a good idea to create it so I created a draft for the moment. Please feel free to review it and propose changes.

File: rts/lra/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# MicroProfile LRA
+
+[MicroProfile LRA](https://github.com/eclipse/microprofile-lra) is a specification of an annotation based API that enables loosely coupled services to coordinate long running activities in such a way as to guarantee a globally consistent outcome without the need to take long duration locks on data.
+
+### MicroProfile Fault Tolerance
+Fault Tolerance provides developers with the following strategies for dealing with failure:
```
Comment (mmusgrov) [inline rts/lra/README.md:6]: We only use the `@Bulkhead` annotation in the code so we cannot use any of `@Timeout`, `@Retry`, `@CircuitBreaker` or `@Fallback` so we should not be documenting them

**Review by mmusgrov — CHANGES_REQUESTED** [review]
Otherwise the PR looks good.

File: rts/lra/README.md (hunk 1 above)
Comment (mmusgrov) [reply rts/lra/README.md:6]: Remark: we use asciidoc in the rest of the LRA module (./rts/lra/test/README.adoc ./rts/lra/API.adoc). What do you think about being consistent and changing the asciidoc files to markdown?

Comment (marcosgopen) [reply rts/lra/README.md]: Thanks @mmusgrov for the review. I updated the README, now it is an asciidoc file.

**Review by mmusgrov — APPROVED** [review]

**Comment by marcosgopen** [comment]
@mmusgrov I added a new test called "testBulkheadRejectionTooManyConcurrentInvocations" that asserts too many concurrent calls throws the bulkhead exception expected. Do you mind reviewing it too? Thanks

**Comment by marcosgopen** [comment]
> @mmusgrov I added a new test called "testBulkheadRejectionTooManyConcurrentInvocations" that asserts too many concurrent calls throws the bulkhead exception expected. Do you mind reviewing it too? Thanks

Sorry for the noise @mmusgrov, I decided not to include the 'testBulkheadRejectionTooManyConcurrentInvocations' test because it sometimes fails. It may fail when there are low resources and the parallel calls are 'simulated' and not real.

**Pull request merged**